app-rails/app/services/auth_service.rb

```diff
@@ -1,8 +1,18 @@
 # frozen_string_literal: true
 
 class AuthService
-  def initialize(auth_adapter = Auth::CognitoAdapter.new)
-    @auth_adapter = auth_adapter
+  def initialize(auth_adapter = nil)
+    @auth_adapter = auth_adapter || default_adapter
+  end
+
+  def default_adapter
+    return Auth::MockAdapter.new if Rails.env.test?
+
+    if Rails.env.development? && ENV["COGNITO_CLIENT_SECRET"].to_s.empty?
```

Comment:
can we add a new environment variable instead like `AUTH_SERVICE` which can have values `"cognito"` or `"local"` or `"mock"` (or just "cognito" and "mock" depending on the answer to my previous question about reusing mock)?